src/IdProviderFactory.php

```diff
@@ -59,13 +59,20 @@ private static function dbExecute() {
 	}
 
 	/**
-	 * Checks whether a WikiPage with the following id/title already exists
+	 * Function to check if string id/title already exists, i.e. is already used as a
+	 * WikiPage (if page already exists)
 	 *
+	 * @param string $text The title or ID to check for uniqueness of the wiki page.
 	 * @return \Closure
 	 */
 	private static function isUniqueId() {
```

Comment:
Naming:

The method name, isUniqueId, suggests it checks uniqueness directly, but the implementation returns a closure. Renaming it to something like getUniqueIdChecker might better reflect its functionality.